### DISCUSSION TITLE
formr maintenance

### DISCUSSION THREAD

**Discussion by cyriltata** [rubenarslan/formr.org#568] [General]
Dear formr users

We are moving the formr database to a faster storage this weekend. As a result, there will be a downtime for formr between **Friday, 30.08.2024 17:00** and **Sunday 01.09.2024 20:00**

Sorry for any inconvenience this might cause and thanks for your understanding.

Best regards
Cyril

**Comment by cyriltata** [comment]
Dear formr users

Unfortunately the data migration is taking longer than expected. You may still not be able to use formr for another 48 hours.
Sorry for any inconvenience and I will inform you once the data migration is done.

best
Cyril

**Reply by xpressio** [reply]
Dear Cyril,

after a brief period of normal operation (somewhere between yesterday, 2024-09-03, 16.00 and today 11.00) I am getting a 503 "service unavailable, data store unavailable" again.

You are probably aware, just wanted to make extra sure ;)

Best
Max

**Comment by jkelgs** [comment]
Dear all,
Since the server update, many participants have reported an issue where they are redirected to the service message after receiving a personal link, as the system no longer recognizes them (notably, all participants have been created before the server update).

While asking them to use a private browser window or clear cookies and cache usually solves the problem, some participants find this difficult.

Is there an easier solution to this issue, without needing participants to switch browsers or clear their browsing history?

Thank you & best regards,
Fabian

**Comment by cyriltata** [comment]
Dear Fabian

Please do you have a test participant whose personal link you can share
with me and I debug this. Normally the update should not affect this.

best
Cyril

**Reply by jkelgs** [reply]
Dear Cyril,
Unfortunately, we have never been able to replicate the issue on our own devices – but I can send you the personal link's of real participants with the issue via e-mail.
Best,
Fabian
